docs/metrics/perplexity.md

```diff
@@ -0,0 +1,118 @@
+---
+search:
+  exclude: true
+---
+
+# Perplexity
+
+!!! info inline end "Requirements"
+
+    It is important to note that perplexity can only be calculated using autoregressive language models.
```

Comment:
What is an "autoregressive language model"? Do we define this anywhere? If not, is there a standard definition (e.g. Wikipedia) that we can link to?